### DISCUSSION TITLE
Uninstall

### DISCUSSION THREAD

**Discussion by CyayC8** [NiyaShy/XB1ControllerBatteryIndicator#83] [Q&A] [answered]
Hello, 

Since it doesn't work with BT I would like to uninstall it. 

There is no uninstall in the folder and it's not listed between my programs in control panel. 

How can I delete?

**Comment by NiyaShy** [comment, marked as answer]
Hey,
the program has no installer and runs stand-alone from the folder you unpacked it to, so if you don't want to use it any longer, close it if it's still running in the systray and then simply delete the folder. The only thing that will be left on your system will be a tiny configuration file within your user profile, and the start menu entry in the "all programs" list (which you can also delete).

**Reply by CyayC8** [reply]
Thank you for this info
